testsuite/tests/basic/boxedints.ml

```diff
@@ -114,18 +113,15 @@ struct
     test 10 (of_string "0x80000000") min_int;
     test 11 (of_string "0xFFFFFFFF") minus_one;
 
-    testing_function "to_string, format";
+    testing_function "to_string";
     List.iter (fun (n, s) -> test n (to_string (of_string s)) s)
       [1, "0"; 2, "123"; 3, "-456"; 4, "1234567890";
        5, "1073741824"; 6, "2147483647"; 7, "-2147483648"];
-    List.iter (fun (n, s) -> test n (format "0x%X" (of_string s)) s)
-      [8, "0x0"; 9, "0x123"; 10, "0xABCDEF"; 11, "0x12345678";
-       12, "0x7FFFFFFF"; 13, "0x80000000"; 14, "0xFFFFFFFF"];
```

Comment:
It would be nice to keep these tests (they exercise conversions from hex strings to integers, not just from integers to hex strings) but make them use Printf.sprintf instead.

Reply:
Good idea. I pushed a commit moving these tests to `lib-printf/tprintf.ml`.